vllm/v1/engine/core.py

```diff
@@ -195,20 +209,12 @@ def signal_handler(signum, frame):
         signal.signal(signal.SIGTERM, signal_handler)
         signal.signal(signal.SIGINT, signal_handler)
 
-        parent_process = psutil.Process().parent()
-        engine_core = None
+        engine_core = EngineCoreProc(*args, **kwargs)
```

Comment:
Note for reviewer:
- `EngineCoreProc` sends a `FAILED` message over the `ready_pipe` if issue during startup
- This is why it is outside of the try ... catch block

Reply:
would be good to leave this in a comment in the code